P5/Source/Specs/att.lexicographic.xml

```diff
@@ -79,6 +86,7 @@ $Id$
       <datatype><dataRef key="teidata.pointer"/></datatype>
     </attDef>
     <attDef ident="mergedIn" usage="opt">
+      <gloss xml:lang="en" versionDate="2009-05-28">mergedIn</gloss>
```

Comment:
Please check and replace all camelCase notations

Reply:
Oh dear. Good point.

There are 272 elements (in the TEI namespace) that have an `@ident` attribute and a `<gloss>` child, but do **not** have a `<gloss>` child that is in English. Those are our targets. We (well, I) assumed that there would be no camelCase values in that list, as those are supposed to have English glosses. However, taking that list and looking for any which match a capital letter reveals 28 that need to be thought about:
~~~~~
elementSpec/@ident=tagUsage
valItem/@ident=soCalled
attDef/@ident=targetEnd
elementSpec/@ident=teiCorpus
elementSpec/@ident=floatingText
elementSpec/@ident=titlePart
attDef/@ident=mergedIn
elementSpec/@ident=superEntry
elementSpec/@ident=locusGrp
elementSpec/@ident=objectType
elementSpec/@ident=finalRubric
elementSpec/@ident=objectDesc
attDef/@ident=ruledLines
attDef/@ident=writtenLines
elementSpec/@ident=typeDesc
elementSpec/@ident=typeNote
elementSpec/@ident=musicNotation
elementSpec/@ident=handShift
elementSpec/@ident=lacunaStart
elementSpec/@ident=lacunaEnd
elementSpec/@ident=variantEncoding
elementSpec/@ident=roleName
elementSpec/@ident=placeName
valItem/@ident=XHTML
valItem/@ident=XML
valItem/@ident=XI
elementSpec/@ident=attList
~~~~~
Since we intend to change the stylesheets so that an English `<gloss>` that matches the `@ident` would be suprressed (i.e., would not appear immediately following the identifier in parentheses), I’m thinking we can use the “should a gloss appear in parens here” as a rule of thumb to help guide decisions on what to do with the above.

Since there at most 28 of them, my plan is to change those that need new `<gloss>` elements that contain other than the value of `@ident` by hand. I will post a list or table of proposed changes here, first.

Reply:
Table of proposed new glosses.

| construct | new EN gloss |
| --------- | ------------ |
| `<tagUsage>` | element usage |
| "soCalled" | so called |
| `@targetEnd` | target end |
| `<teiCorpus>` | TEI corpus |
| `<floatingText>` | floating text |
| `<titlePart>` | title part |
| `@mergedIn` | merged into |
| `<superEntry>` | super entry |
| `<locusGrp>` | locus group |
| `<objectType>` | object type |
| `<finalRubric>` | final rubric |
| `<objectDesc>` | object description |
| `@ruledLines` | ruled lines |
| `@writtenLines` | written lines |
| `<typeDesc>` | typeface description |
| `<typeNote>` | typographic note |
| `<musicNotation>` | music notation |
| `<handShift>` | handwriting shift |
| `<lacunaStart>` | lacuna start |
| `<lacunaEnd>` | lacuna end |
| `<variantEncoding>` | varient encodings |
| `<roleName>` | role name |
| `<placeName>` | place name |
| "XHTML" | XHTML |
| "XML" | XML |
| "XI" | XI |
| `<attList>` | attribute list |

Reply:
@sydb Just left a tiny proof correction in passing: varient to variant

Reply:
Thank you, @ebeshero !

Reply:
OK. Have addressed 24 of ’em  from the above table (leaving XHTML, XML, and XI alone), plus `<soCalled>` which, for some reason, is not in the above table. Only difference from above table is I used “variant encoding” singular for `<variantEncoding>` (not sure what I was thinking when I made it plural in table, above).

Note that although 25 were changed, only 24 files were changed because 2 of them (`@ruledLines` and `@writtenLines`) are defined in the same file.

Change was dee5c69.